### PR TITLE
add data-audiotype

### PR DESCRIPTION
**Pull request by kdimatteo**
drop dist/test.html

### CODE REVIEW

File: dist/jquery.audio-plugin.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
     this.volume       = this.$el.find('.volume');
     this.audioURL     = this.$el.data('audiourl');
     this.song         = new Audio(this.audioURL);
-    this.song.type    = 'audio/mpeg';
+    this.song.type    = this.$el.data('audiotype') || 'audio/mp3';
```
Comment (kdimatteo) [inline dist/jquery.audio-plugin.js]: @goyalvarun601

Comment (goyalvarun601) [inline dist/jquery.audio-plugin.js]: is 'audio/mp3' a valid type? Shouldn't it be 'audio/mpeg3' ?

Comment (kdimatteo) [inline dist/jquery.audio-plugin.js]: yes, per : https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/CreatingContentforSafarioniPhone/CreatingContentforSafarioniPhone.html

Comment (goyalvarun601) [inline dist/jquery.audio-plugin.js]: ok :+1:

**Pull request merged**